packages/block-library/src/query-pagination-next/edit.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, RichText } from '@wordpress/block-editor';
+
+export default function QueryPaginationNextEdit( {
+	attributes: { label },
+	setAttributes,
+} ) {
+	const placeholder = __( 'Next Page' );
+	return (
+		<>
+			<div { ...useBlockProps() }>
```

Comment:
Is this div also output in the frontend? if not, we should remove it and use the RichText as the wrapper (see Paragraph for instance)

Reply:
(Maybe it's not possible because "a" is inline by default though). @ellatrix do you know?

Reply:
>Is this div also output in the frontend?

It's not. I'm not sure about this though:
>Maybe it's not possible because "a" is inline by default though

Reply:
It is possible, we just need to update the display property to `inline-block` for example.